src/components/HighTable/HighTable.module.css

```diff
@@ -63,10 +66,10 @@
   /* column resize */
   thead [role="separator"] {
     position: absolute;
-    top: 0;
+    top: 1px;
```

Comment:
why did this need to change?

Reply:
it was to ensure the separator is visually inside the cell header